flake.nix

```diff
@@ -399,6 +399,7 @@
                 (baseNameOf path) [ "server" "doc" ]
               );
           };
+          withChromium = false;
```

Comment:
This needs to go into `nix/default.nix`'s `shellFor` instead. Add `withChromium` to the list of args: 
```
  shellFor =
    { packages ? [ ]
    , inputsFrom ? [ ]
    , shellHook ? ""
    , formatter ? "purs-tidy"
    , pursls ? true
    , nodeModules ? projectNodeModules
    , packageLockOnly ? false
    , withChromium ? false                 # here
    }:
```
Then add the suggestion below to the `pkgs.mkShell`'s `buildInputs` instead of the `packages` of our own `devShell`